app/views/users/show.html.erb

```diff
@@ -12,7 +12,7 @@
     <% if display_repos != nil %>
       <section class="repos">
         <ul>
-          <% display_repos.each do |repo| %>
+          <% display_repos.first(5).each do |repo| %>
```

Comment:
I think we may be able to refactor the display_repos method so that it's only passing 5 to the view & then this logic can be removed.